modules/jdbc-test/src/test/java/org/testcontainers/jdbc/JDBCDriverTest.java

```diff
@@ -199,7 +198,7 @@ private HikariDataSource verifyCharacterSet(String jdbcUrl) throws SQLException
         boolean result = new QueryRunner(dataSource).query("SHOW VARIABLES LIKE 'character\\_set\\_connection'", rs -> {
             rs.next();
             String resultSetInt = rs.getString(2);
-            assertEquals("Passing query parameters to set DB connection encoding is successful", "utf8", resultSetInt);
+            assertEquals("Passing query parameters to set DB connection encoding is successful", "utf8mb4", resultSetInt);
```

Comment:
This is to match the changes mentioned [here](https://dev.mysql.com/doc/connector-j/8.0/en/connector-j-reference-charsets.html)
> For Connector/J 8.0.13 and later:

> When UTF-8 is used for characterEncoding in the connection string, it maps to the MySQL character set name utf8mb4.